llvm/test/Transforms/LoopUnroll/unroll-calls-if-forced.ll

```diff
@@ -0,0 +1,35 @@
+; RUN: opt -passes=loop-unroll -disable-verify -S < %s | FileCheck %s
```

Comment:
Why does this `-disable-verify`?

Please also use UTC.

Reply:
> Why does this -disable-verify?

Copy-paste from another test; I removed this.

> Please also use UTC.

I assume for commit timestamps? Done.

Reply:
UTC as in llvm/utils/update_test_checks.py to automatically generate CHECK lines

Reply:
Ah, that makes more sense. Thanks